config/game_options.txt

```diff
@@ -239,7 +239,7 @@ DEFAULT_LAWS 0
 ## See datums\ai_laws.dm for the full law lists
 ## IE, SPECIFIED_LAWS asimovpp, SPECIFIED_LAWS robocop, SPECIFIED_LAWS antimov
 
-SPECIFIED_LAWS asimovpp
```

Comment:
Revert everything that touches the config

Reply:
it was already in an edit but ok

Reply:
already done =/= good